bin/flutter.bat

```diff
@@ -88,7 +88,10 @@ GOTO :after_subroutine
 
   :do_sdk_update_and_snapshot
     ECHO Checking Dart SDK version...
-    CALL PowerShell.exe -ExecutionPolicy Bypass -Command "& '%FLUTTER_ROOT%/bin/internal/update_dart_sdk.ps1'"
+    SET update_dart_bin="%FLUTTER_ROOT%/bin/internal/update_dart_sdk.ps1"
```

Comment:
Do you really need double quotes on RHS?

Reply:
I _think_ so in case there are spaces in the path.  I cribbed this code off of SO...

Reply:
>I think so in case there are spaces in the path. I cribbed this code off of SO...

Generally speaking when setting variable to a value with spaces you don't need quotes. When you do use quotes in `set` they will become part of the variable's value. That could have been your intent. Or not :-)

Reply:
I'm wondering whether the difference is substantive - I tried the fix in my Windows 10 instance, and it worked.  Do you think it'd preferable to remove the quotes?  If so, I'll try that and verify that it still works.

Reply:
My preference is to not to use quotes unless there is a reason to use them.

Reply:
Done